omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -114,6 +114,49 @@ public void shouldEditAPatientIdentifier() throws Exception {
 	}
 	
 	@Test
+	public void shouldEditAPatientIdentifierWithPreferredAttibuite() throws Exception {
+		PatientIdentifier patientIdentifier = service.getPatientIdentifierByUuid(getUuid());
+		Boolean originalPreferredStatus = patientIdentifier.getPreferred();
+		Boolean postPreferredStatus = !originalPreferredStatus;
```

Comment:
What is the use of the above variable?

Reply:
The identifier to be changed. I guess I should change the name